config/totem.conf.example

```diff
@@ -50,6 +50,10 @@ totem {
       uri = ["http://127.0.0.1:7703/metadata/", "http://127.0.0.1:7707/metadata/"]
       resultRoutingKey = "metadata.result.static.totem"
     }
+    office_meta {
+      uri = ["http://127.0.0.1:7730/office_meta/"]
+      resultRoutingKey = "office_meta.result.static.totem"
```

Comment:
Please change office_meta and office_meta.result.static.totem to officemeta so the styles match.